src/explore-education-statistics-frontend/src/modules/find-statistics/FindStatisticsPageNew.tsx

```diff
@@ -0,0 +1,155 @@
+                    target="_blank"
+                  >
+                    Educational statistics for Northern Ireland
+                  </a>
+                </li>
+                <li>
+                  <a
+                    href="https://www.gov.scot/statistics-and-research/?cat=filter&amp;topics=Education"
+                    rel="noopener noreferrer"
+                    target="_blank"
+                  >
+                    Educational statistics for Scotland
+                  </a>
+                </li>
+                <li>
+                  <a
+                    href="https://statswales.gov.wales/Catalogue/Education-and-Skills"
+                    rel="noopener noreferrer"
+                    target="_blank"
+                  >
+                    Educational statistics for Wales
+                  </a>
+                </li>
+              </ul>
+            </RelatedInformation>
+          </div>
+        </div>
+        <div className="govuk-grid-row">
+          <div className="govuk-grid-column-one-third">
+            Search and filters here
+          </div>
+          <div className="govuk-grid-column-two-thirds">
+            <LoadingSpinner loading={isLoading}>
+              <h2
+                className="govuk-!-margin-bottom-2"
+                aria-live="polite"
+                aria-atomic="true"
+              >
+                {totalResults} {totalResults !== 1 ? 'results' : 'result'}
+              </h2>
+
+              <a href="#searchResults" className="govuk-skip-link ">
+                Skip to search results
+              </a>
+              <p
+                aria-live="polite"
+                aria-atomic="true"
+                className="govuk-!-margin-top-1"
+              >{`Page ${currentPage} of ${paging.totalPages}, showing all publications`}</p>
+
+              {/* TODO EES-3517
+             <p className="govuk-visually-hidden">
+              Sorted by newest publications
+            </p>
+            <p>Current filters here</p>
+            <p>Sorting controls here</p>*/}
+
+              <hr />
+
+              {/* TODO EES-3517 show different message if search / filter returns no results  */}
+              {currentPublications.length === 0 ? (
+                <div className="govuk-inset-text" id="searchResults">
+                  No data currently published.
+                </div>
+              ) : (
+                <ul className="govuk-list" id="searchResults">
+                  {currentPublications.map(publication => (
+                    <PublicationSummary
+                      key={publication.id}
+                      publication={publication}
+                    />
+                  ))}
+                </ul>
+              )}
+
+              <Pagination
+                baseUrl="/find-statistics"
+                queryParams={{ newDesign: true }} // TO DO EES-3517 make sure params for filters / search are included here
```

Comment:
We should be able to remove these props if we implement Next router-based defaults as suggested in an earlier comment.